pkg/cmd/cli/restore/delete.go

```diff
@@ -17,31 +17,190 @@ limitations under the License.
 package restore
 
 import (
+	"bufio"
 	"fmt"
+	"os"
+	"strings"
 
+	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
 
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	kubeerrs "k8s.io/apimachinery/pkg/util/errors"
+
+	arkv1api "github.com/heptio/ark/pkg/apis/ark/v1"
 	"github.com/heptio/ark/pkg/client"
 	"github.com/heptio/ark/pkg/cmd"
+	"github.com/heptio/ark/pkg/cmd/util/flag"
+	clientset "github.com/heptio/ark/pkg/generated/clientset/versioned"
 )
 
+// NewDeleteCommand creates and returns a new cobra command for deleting restores.
 func NewDeleteCommand(f client.Factory, use string) *cobra.Command {
+	o := &DeleteOptions{}
 	c := &cobra.Command{
-		Use:   fmt.Sprintf("%s NAME", use),
-		Short: "Delete a restore",
-		Args:  cobra.ExactArgs(1),
-		Run: func(c *cobra.Command, args []string) {
-			arkClient, err := f.Client()
-			cmd.CheckError(err)
+		Use:   fmt.Sprintf("%s [NAMES]", use),
+		Short: "Delete restores",
+		Example: `# delete a restore named "restore-1"
+	ark restore delete restore-1
+
+	# delete a restore named "restore-1" without prompting for confirmation
+	ark restore delete restore-1 --confirm
 
-			name := args[0]
+	# delete restores named "restore-1" and "restore-2"
+	ark restore delete restore-1 restore-2
 
-			err = arkClient.ArkV1().Restores(f.Namespace()).Delete(name, nil)
-			cmd.CheckError(err)
+	# delete all restores triggered by schedule "schedule-1"
```

Comment:
this example isn't quite appropriate for restores since they aren't generally labeled with `ark-schedule=<foo>`. I'd change this example to just say "delete all restores with label foo=bar", and update the selector used too.